docs/guide/database-insert-update.md

```diff
@@ -78,7 +92,7 @@ In the example above, `self` refers to the `dog` piece type module since that is
 
 Updating pages works the same way as pieces with the same arguments to the `update` method. There are a few things to keep in mind when working on pages, however.
 
-**We often call the page `update` method from `self.apos.page`.** `self.update` does work from individual page type modules and sometimes that is the right way to call it. But since pages can change their `type` property (unlike pieces) they share a single `update` method from the `@apostrophecms/page` module. `self.update` on individual page type modules is simply a wrapper that method.
+**We often call the page `update` method from `self.apos.page`.** `self.update` does work from individual page type modules and sometimes that is the right way to call it. But since pages can change their `type` property (unlike pieces) they share a single `update` method from the `@apostrophecms/page` module. `self.update` on individual page type modules is simply a wrapper around that method.
 
 **There is a better way to move pages within the [page tree](/guide/pages.md#connecting-pages-with-page-tree-navigation).** The `@apostrophecms/page` module has a dedicated `move()` method for that purpose. Documentation of that method is upcoming. In the meantime, refer to the [REST API information about page tree placement](/reference/api/pages.md#post-api-v1-apostrophecms-page).
```

Comment:
It's not a better way, it's the only way at the JS api level (update can't do it).

Reply:
I was thinking that it was possible with a mess of extra steps, but it's an unnecessary mystery.  👍